packages/calcite-components/tailwind.config.ts

```diff
@@ -17,6 +19,27 @@ const config: Config = {
       },
     },
   },
+  plugins: [
+    plugin(({ addUtilities }) => {
+      addUtilities({
+        // we override preset focus utils to allow using the internal focus color
+        ".focus-base": {
+          "outline-color": "transparent",
+        },
+        ".focus-normal": {
+          outline: "2px solid var(--calcite-internal-color-focus)",
```

Comment:
should we use a var for `2px`? Maybe the border width var? var(--calcite-spacing-base)